src/rhsm/connection.py

```diff
@@ -425,7 +425,7 @@ def __init__(self, code: int, msg: str, deleted_id: Any):
         self.deleted_id = deleted_id
 
 
-class NetworkException(ConnectionException):
+class UnknownContentException(ConnectionException):
```

Comment:
... using so many commits can be confusing.

Reply:
> It looks like that this commit only rename class method, but the message of exception is still the same. It doesn't say anything about unknown/empty content. Do I miss anything there?

No, it is a commit that only renames that exception, explaining why in its commit message. Since there are some uses of it around, it was cleaner to me to make sure all is grouped together.